### PR TITLE
Restructure and add to examples

### PR DESCRIPTION
**Pull request by harmtemolder**
I've split up the `interception-pipe` configurations into separate subfolders and added explanations for each to the `README.md` to better explain how to work with them. Also added a couple of extra examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,81 @@
...
 
-User configuration files can be placed under subdirectories of `config`
-directory. Executables will be generated per subdirectory, so you can easily
-manage your rules if you need more instances of `interception-k2k`.
+User configuration files can be placed under subdirectories of `examples` directory, or in a new dir next to it. Executables will be generated per subdirectory, so you can easily manage your rules if you need more instances of `interception-k2k`.
+
+- For simple 1-to-1 mappings, use `map-rules.h.in`.
+  - If you want to disable a key, map it to `KEY_RESERVED`.
+- For many-to-1, use `multi-rules.h.in`.
+- Note that there does not seem to be a way to map a single key input to output multiple keys. Use dual-function-keys for that.
```
Comment (zsugabubus) [inline README.md]: Could you rephrase it to something like "There is no way..."?

File: README.md [hunk 2]
```diff
@@ -1,26 +1,81 @@
...
+
+This remaps the right half of an ANSI laptop keyboard after you have moved around some of its keycaps for a WideSym mod (inspired by [DreymaR's Colemak-CAWS](https://dreymar.colemak.org/ergo-mods.html#symbols)).
+
+### shift2caps
+
+This toggles <kbd>caps lock</kbd> when both <kbd>shift</kbd> keys are pressed together.
+
+### vim-overlay
+
+Holding <kbd>e</kbd> activates vim-like functions on the right side of the keyboard, and holding <kbd>i</kbd> activates some on the left. Note that this mapping seems to be incorrect, so it would need some work before these are actual vim keys. The correct ones look to be commented out.
```
Comment (zsugabubus) [inline README.md]: > Note that this mapping seems to be incorrect…

Those keys are based on the Dvorak layout so could please mention it instead of saying incorrect?

I'm not sure but maybe it could be renamed to `vim-overlay-dvorak` and/or make a "correct" `vim-overlay-qwert?` copy of it. (Please don't waste your time with the latter if you personally don't use it.)

**Review by zsugabubus — CHANGES_REQUESTED** [review]
Thanks a million for your work!

Apart from those two little changes could you please squash `Fix HTML tags` and `Do not use vim overlay in the example udevmon.yaml` as they both seem to be fixup commits. More or less. But to be honest I'm fine with it as is so please do it only if it takes less than a few minutes.

**Comment by harmtemolder** [comment]
I made the changes and squashed `Fix HTML tags` into the commit before. But if you want to keep the commits in your repo to a minimum, feel free to squash this PR into a single commit when merging.

**Comment by harmtemolder** [comment]
Note that this PR also fixes #11

**Comment by zsugabubus** [comment]
`make` fails on `KEY_g` that makes we wonder whether a non-Dvorak `vim-overlay` example really makes sense. I guess no because the keys are mapped to be convenient for a Dvorak user (or at least just for me) and I see no value in it as an extra example. So please restore the original example, maybe rename it to `vim-overlay-dvorak`, and just mention in the README that it is for Dvorak layout.

I've looked into the commits again and it would make sense for me to squash all into one, except the one that adds `qwerty-ws`. This commit contains an unrelated README change, please move it to the other commit.

**Comment by harmtemolder** [comment]
What I did for the non-dvorak vim overlay was just use the commented keys, but that didn't work because there is a lowercase `g` there:

```
    { TAP(KEY_VIML), .action_key = KEY_g,         .hold_key = KEY_HOME,      .tap_typing = 1 }, \
    { TAP(KEY_VIML), .action_key = KEY_G,         .hold_key = KEY_END,       .tap_typing = 1 }, \
```

As an example I agree that it's enough to have the dvorak one. People can then adjust for themselves.

I'll squash and split the commits and remove the vim overlay

**Comment by zsugabubus** [comment]
Thanks again for your work!
